### DISCUSSION TITLE
Unmarked SOIC-8 Chip

### DISCUSSION THREAD

**Discussion by tetatdo** [ludwig-v/wireless-carplay-dongle-reverse-engineering#86] [General]
So I have the autokit version of this device. I think hardware wise its identical to the u2w. When I saw the unmarked SOIC-8 IC, it looked like a SPI flash chip to me, so I decided if i could see if i could dump it. Long story short, I tried to unsolder pin1 because my tl-866 always flips out when you try to read spi flash in circuit..... Well I destroyed pin 1...badly.

Tried using the unit and it seemingly works like nothing happened. 

Thanks to here, I now understand the rom resides on the wsop-8 macronix flash located next to this unmarked chip. But without that thing properly connected, it seems to work ok. I even flashed it to a new version, and verified my phone connects to carplay and works just fine. Wonder what that thing does? 

in order to try to save the IC, i tried to scrape some of the packaging away to see if i can find any of the remaining lead so I can solder it back and get a connect back to pin 1 from the remaining lead. I gave up when I tested and realized all is working.
![signal-2021-06-14-225109](https://user-images.githubusercontent.com/85921993/122000087-1dc77780-cd63-11eb-89ec-4ddb73a29797.jpeg)

**Comment by iDoka** [comment]
My assumption based on PCB topology near mentioned SOIC-8:
pin1 (also as remaining pin2,3,4) tied to GND polygon (probably power IC?) and pins 1..4 have redundancy functionality for some reason (enchancement thermal dissipation, decreare wire resistance for high current net for example).

**Reply by tetatdo** [reply]
youre right. after i posted it, i realized i attacked the wrong pin. I thought it was VCC, but its not. I wonder what that pin does. Maybe its just a chip select line thats stuck high or low (depending on if its active low or not).


In any case. interesting. I'm half tempted to take the entire chip off with a hot air station and see if it will remain working.

wonder what that chip does.
